src/components/recommendation/recommendation.jsx

```diff
@@ -0,0 +1,29 @@
+import PaintingTile from '../paintings-page/painting-tile/painting-tile';
+import styles from './recommendation.module.scss';
+
+export default function Recommendation({ collectionData = [] }) {
+  const paintings = collectionData.paintingsCollection
+    ? collectionData.paintingsCollection
+    : collectionData;
+
+  const classNames = {
```

Comment:
The fact that we have 2 different implementation using the same common prop named `classNames` across many components is becoming confusing. I think I've mentioned this before, when you pass in classNames as an object, make it clear that it's an object as opposed to a string.